### DISCUSSION TITLE
💡 شماره ویژه

### DISCUSSION THREAD

**Discussion by ohmydevops** [softwaretalks/newsletter#132] [Ideas]
<div dir='rtl'>

### شماره ویژه چیست؟

در حال حاضر، هر هفته خبرنامه‌ای شامل محتوایی که کاربران ارسال کرده‌اند، ایمیل میکنیم. این هسته و مهم‌ترین ارزش خبرنامه Softwaretalks میباشد.

داشتم فکر میکردم که چطور میتونیم گاهی تنوع و خلاقیت‌هایی به خبرنامه اضافه کنیم. و اسم اون شماره‌های خاص رو شماره ویژه بزاریم. شماره‌ای که علاوه بر محتوای همیشگی خبرنامه، محتواهای جذاب ولی متفاوتی برای خوندن و استفاده کردن داره.  

این اتفاق میتونه هر ماه مثلا رخ بده یا هر وقت که شرایطش برقرار شد. فعلا ایده‌ای ندارم که چطور قراره بشه و اصلا قراره چه کار کنیم! این بحث رو از این جهت شروع کردم که اگر کسی تمایل داشت ایده بده، بتونه اینجا نظرشو بنویسه و خودمم فکرهامو بلند بلند اینجا یادداشت کنم تا به نتیجه مشخصی برسم

 ---

در اینجا، ایده‌هایی که میشه درباره شماره ویژه داد رو مطرح میکنیم.

</div>

**Comment by ghost** [comment]
منظورتون از شمارهٔ ویژه چیه؟

**Reply by ohmydevops** [reply]
فعلا فقط یادداشتش کرده بودم که امشب که رسیدم منزل تکمیلش کنم. الان پست اصلی رو آپدیت میکنم و توضیحات رو میزارم

**Reply by ohmydevops** [reply]
توضیحات رو تکمیل کردم جناب کوکبی

**Comment by ohmydevops** [comment]
<div dir='rtl'>

### آسان‌سازی فرایند مشارکت در پروژه‌های منبع‌باز

اولین پیشنهادی که چند وقتیه بهش فکر میکنم، معرفی پروژه‌هایی منبع‌باز برای مشارکت کردنه. به این صورت که کاربران اگر روی پروژه‌ای کار میکنند و دوست‌ دارند فردی بیاد و روی بخشی از کار، کمکشون کنه، بتونن از تریبون خبرنامه استفاده کنن و پروژه‌شونو معرفی کنن و بگن که چطور و برای چه کارهایی نیاز به کمک دارند. یا این که مثلا اگر کسی داره از پروژه منبع‌باز خاصی استفاده میکنه و به هر دلیلی باگی پیدا میکنه یا مشکلی هست که نمیتونه خودش حل کنه، کافیه اینجا معرفیش کنه تا اگر کسی توانایی داشت بتونه در اون پروژه مشارکت کنه. در کل هدف از این موضوع، راحت‌تر کردن فرایند پیداکردن پروژه برای مشارکت کردنه. چه پروژه‌های جامعه فارسی‌زبان چه هر پروژه‌ دیگه‌ای.
</div>

**Reply by ghost** [reply]
ایدهٔ خوبیه ولی فکر نکنم موضوعی باشه که یک شماره رو کامل بهش اختصاص داد؛ ولی می‌شه در هر شماره یک پروژه معرفی کرد. الان همین خبرنامه ۴۰۰ تا عضو داره و فرایند مشارکتش هم در حد یک لینک دادنه ولی پرنده پر نمی‌زنه. حتی تو Discussion خبری نیست!

**Reply by ohmydevops** [reply]
<div dir='rtl'>
بله دقیقا. شماره ویژه یعنی شماره‌ای که علاوه بر محتوای اصلی، محتواهایی از این جنس هم داخلش پیدا میشه که هر‌از‌چند‌گاهی احتمالا باشه.
</div>

**Reply by ohmydevops** [reply]
<div dir='rtl'>
چقدر بده که گیتهاب فارسی رو دیفالت RTL نشون نمیده. دائم باید تگ div بزارم. فکر میکنم نیازه یک راه‌کار خودکار براش پیدا کنم
</div>

**Reply by ghost** [reply]
<div dir="rtl">
آی گفتی!

یعنی واقعاً اضافه کردن یه دکمهٔ ساده به نوار ابزار اینقدر سخته؟ فقط گیتهاب هم نیست؛ VSCode هم راست به چپ نداره؛ سابلایم که کلاً کاراکتر فارسی پشتیبانی نمی‌کنه و ... . بحث Layout هم نیست بگیم زمان می‌بره.

من برای راست‌چین کردن متن‌ها از [فونت‌آرا ](https://mimalef70.github.io/fontara) استفاده می‌کنم؛ ولی چون همه این افزونه رو ندارن بازم مجبورم div rtl بزارم. الان یه افزونه‌ای پیدا کردم این کار رو به صورت خودکار انجام می‌ده. فقط یه [مشکل کوچیک](https://github.com/amir-yaghoubi/RTL-markdown/issues/3) داره. اگه بتونید این مشکل رو حل کنید و توی گوگل وب استور بذارید عالی می‌شه. اگه جاوااسکریپتم خوب بود همین امشب مشکلش رو حل می‌کردم.
</div>

**Reply by ohmydevops** [reply]
منم چون فونت‌آرا هستم همیشه این عذاب‌وجدان رو دارم که اگر div نزارم بقیه متن رو سخت باید بخونن.

**Reply by ohmydevops** [reply]
افزونه جالبی به‌نظر میاد. سعی میکنم بررسی کنم اگر فرصت داشتم منم کمک میکنم

**Comment by ghost** [comment]
<div dir="rtl">
اجازه بدید منم بلند فکر کنم.
شمارهٔ ویژه رو به نظرم باید اختصاص داد به مطالبی که در جستجوهای معمول کمتر باهاشون برخورد می‌کنیم. این مطالب اغلب از جنس تجربه و مربوط به بازار ایران هستند.

برای مثال:
- مطالب تألیفی (که اغلب به تجربیات افراد برمی‌گرده) مثلاً [از هزاران درخواست در روز به هزاران درخواست در ثانیه](https://vrgl.ir/JPs0y)

- تجربیات کار در شرکت‌ها (خصوصاً شرکت‌های داخلی) مثلاً [۱۶ درسی که از کار کردن در عکس‌پرینت آموختم](https://virgool.io/@mahdi/%DB%B1%DB%B6-%D8%AF%D8%B1%D8%B3-%D9%85%D9%87%D9%85%DB%8C-%DA%A9%D9%87-%D8%A7%D8%B2-%DA%A9%D8%A7%D8%B1-%DA%A9%D8%B1%D8%AF%D9%86-%D8%AF%D8%B1-%D8%B9%DA%A9%D8%B3%D9%BE%D8%B1%DB%8C%D9%86%D8%AA-%D8%A2%D9%85%D9%88%D8%AE%D8%AA%D9%85-tqghmkyddjgv)

- گزارش‌های سالیانه و ماهیانه (کافه‌بازار، دیوار، جاب‌ویژن و ...)
- آموزش‌های جامع خاص یا بررسی‌های تحلیلی جذاب مثل [داستان طراحی موبایل‌بانک تجارت](https://virgool.io/RedAcademy/tejarat-dcvu3yzltigl)
- تجربیات کار با شرکت‌های خارجی و نقد کردن پول مثلاً [فریلنسر دات کام](https://virgool.io/@ar4sh/freelancer-weqqmxl16jig) یا [چطور درآمد یوتیوب را دریافت کنیم؟](https://www.youtube.com/watch?v=G6oP1-i5w4I)

از طرفی چون پیدا کردن این نوع مطالب راحت نیست نمی‌شه برای شمارهٔ ویژه به یک بازهٔ زمانی ماهیانه پایبند بود. شاید این ایده بد نباشه که شمارهٔ ویژه رو اختصاص بدیم به بهترین مطالب هفته‌نامه‌ها و مابینش یا ذیل یک عنوان دیگه در خبرنامه، مطالب خاصی اگه داشتیم بذاریم (مثل BONUS Lecture که تو بعضی از آموزش‌های خارجی هست). 

البته اینجا بازم سرِ این بحث هست که انتخاب یک مطلب از هفته‌نامه بر چه اساسی باشه؟ یا اصلاً چرا فلان مطالب باید ویژه باشه. خلاصه یکم بحث سلیقه‌ای می‌شه.
</div>

**Reply by ohmydevops** [reply]
<div dir='rtl'>

البته الان در حال حاضر محدودیتی از لحاظ اشتراک گذاشتن محتوا نداریم. یعنی  همه این نوع محتواها که بالا ذکر کردید میتونه توی خبرنامه معمولی هم قرار بگیره.

البته این که سرجمع، مرتب و گلچین‌شده در اختیار کاربر قرار بدیم میتونه خودش یک ارزش قابل تامل باشه. 
</div>

**Reply by ghost** [reply]
<div dir="rtl">
خب اینجوری هر نوع محتوای دیگه‌ای اعم از محتوایی که خودتون پیشنهاد دادید می‌تونه کاندیدایی برای قرار گرفتن در خبرنامه باشه. با این حساب دوباره این سوال پیش می‌آد که شمارهٔ ویژه چیه؟ 😊
</div>

**Reply by ohmydevops** [reply]
<div dir='rtl'>

چیزی که توی ذهن من به اسم شماره ویژه وجود داره اینه که از روتین خبرنامه خارج باشه. حالا به هر شکل که شدنیه. مثلا همون موضوعی که گفتم بالاتر، معرفی ۳ یا ۴ پروژه ایرانی/غیرایرانی برای شروع مشارکت کاربران تازه‌کار در پروژه‌های منبع‌باز. یا به قول شما، گردآوری مجموعه‌هایی برای کاربر.
من چند روز آینده که کارهام کمتر بشه از سر فرصت روی این موضوع کار میکنم و سعی میکنم اولین شماره در سال جدید رو که میشه ۱۱اُمین شماره، با همین ساختار پیش ببرم؛ یعنی خبرنامه + محتواهای شماره ویژه

فکر میکنم نیازه یک شماره اینطوری ارسال بشه تا اون چیزی که توی ذهنم دارم رو بتونم بهتر برای بقیه توصیف کنم.

</div>

**Comment by ohmydevops** [comment]
<div dir='rtl'>

**ضبط ویدیوهایی کوتاه از مهم‌ترین مطالب شماره‌های اخیر**

میشود در یک فایل ویدیویی یا صوتی، محتواهای مهم و جذاب شماره‌های اخیر را بررسی عمیق‌تری کرد.

</div>

**Reply by ghost** [reply]
آقا من که دیگه div rtl نمی‌زنم. خسته شدم. 😀
ایدهٔ خوبیه؛ ولی بررسی عمیق‌تر رو معمولاً در مقالات و کتب انجام می‌دن نه ویدیو و پادکست؛ ولی خب همون بررسی اجمالی و آشنایی اولیه هم چون رسانه غیرمتنی است تأثیرش بیشتره و می‌تونه روی جذب مخاطب و افزایش مشارکت اثرگذار باشه. البته اگه دستِ تنها هستید، ویدیو رو پیشنهاد نمی‌کنم چون خیلی زمان‌بره؛ ولی با یک متن خوب و یک صدای خوب پادکست رو زودتر می‌شه جمع کرد.

برای تدوین پادکست می‌تونید رو من حساب کنید. 
شما فقط صداهای خام ضبط شده رو برام بفرستید.
با Audition کار کردم.

**Comment by ohmydevops** [comment]
<div dir='rtl'>

**پرونده سلامت روانی و جسمی مهندسان نرم‌افزار**

میشود در شماره‌های ویژه روی سلامت جسمی/روانی مهندسان نرم‌افزار و مشکلات مربوط به این رشته و کار پرداخت. هم بررسی هم راه‌حل به کمک متخصص‌های مربوطه.
</div>

**Comment by ohmydevops** [comment]
<div dir='rtl'>

این مدت بهش فکر کردم و به نظرم رسید تنها چیزی که مهمه توی خبرنامه، اینه که یک خبرنامه خوب و مفید برای همیشه بمونه. صرف انرژی برای بخش‌های دیگه خبرنامه شاید باعث بشه تمرکز و وقتم از خود خبرنامه خارج بشه. با توجه به زمان محدودی که دارم به این نتیجه رسیدم فعلا برای شروع و ایجاد تمایز در شماره‌های ویژه، توی کانال Softwaretalks مهم‌ترین/جذاب‌ترین اخبار خبرنامه‌ رو مرور کنم و توشون بیشتر عمیق بشم. یک نسخه باید بسازم تا ببینم این ایده چطوری از آب در میاد

</div>

**Comment by ohmydevops** [comment]
طبق صحبتی که کردیم اگر منابع خوبی درباره اون موضوع پادکست‌های Digg هست میشه اینجا گذاشت یا توی خصوصی توییتر برام ارسال کنین
@Hameds

**Reply by ohmydevops** [reply]
راستی من تایم خالی پیدا کردم و ممنون میشم منابع مرتبط با این موضوع رو برام ارسال کنین. @Hameds

**Reply by Hameds** [reply]
می‌دونم باید زمان‌ها رو با هم هماهنگ کنیم. الان زمان شلوغی کاری من هست ولی ایده‌ای دارم که شاید برای این «شماره ویژه» مناسب باشه. ایمیل می‌زنم.
